using-mobile-extensions/adobe-edge-consent/README.md

```diff
@@ -0,0 +1,162 @@
+# Adobe Experience Platform Edge Consent
+
+## Configure the Adobe Experience Platform Edge Consent extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Consent Collection** extension, and click **Install**.
+3. Set your desired default consent level.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+### Configure the AEP Edge Consent extension
+
+![Adobe Edge Network extension configuration](../../.gitbook/assets/mobile-edge-consent-launch-configuration.png)
+
+
+
+## Add the AEP Edge Consent extension to your app
+
+### Download and import the Edge Consent extension
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+
+   ```java
+   implementation 'com.adobe.marketing.mobile:core:1.+'
+   implementation 'com.adobe.marketing.mobile:lifecycle:1.+'
+   implementation 'com.adobe.marketing.mobile:edge:1.+'
+   implementation 'com.adobe.marketing.mobile.edgeidentity:1.+'
+   implementation 'com.adobe.marketing.mobile.edgeconsent:1.+'
+   ```
+
+2. Import the Mobile Core and Edge extensions in your application class.
+
+   ```java
+    import com.adobe.marketing.mobile.*;
+   ```
+
+   {% endtab %}
+
+{% tab title="iOS" %}
+
+1. Add the Mobile Core and Edge extensions to your project using Cocoapods. Add following pods in your `Podfile`:
+
+   ```swift
+   use_frameworks!
+   target 'YourTargetApp' do
+       pod 'AEPCore'
```

Comment:
Do you need to include AEPServices here too?
Also, for consistency with Android dependencies, you should include AEPSignal.

Reply:
The Android dependencies do not include signal as it is not needed.

AEPServices is fetched automatically from AEPCore: https://github.com/adobe/aepsdk-core-ios/blob/main/AEPCore.podspec#L20